tests/console/journalctl.pm

```diff
@@ -193,7 +194,7 @@ sub run {
         systemctl 'restart systemd-journald.service';
         script_run('cat /dev/ttyS101 > /var/tmp/journal_serial.out & true');
         assert_script_run('echo "journal redirect output started (grep for: aeru4Poh eiDeik5l)" | systemd-cat -p info -t redirect');
-        assert_script_run('grep "aeru4Poh eiDeik5l" /var/tmp/journal_serial.out', fail_message => "Forward to syslog failed");
```

Comment:
Maybe we can use the internal of `script_retry` https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L1518

But that is only sugar syntax, this way it gives more hints to the reviewer, but the code would look better on the other hand,

Reply:
I had the same thought but decided that it might be better to give a meaningful error message. `script_retry` unfortunately doesn't support a `fail_message` yet.

I think a meaningful error message is more important here, as this is helpful to the developer and the reviewers. What do you think?

Reply:
Agreed and I have no preference at all. But maybe the forwarding info to `syslog` might be misleading as `rsyslog` is not present due to conflict with `systemd-logger` package ( provides `/var/log/journal` dir )

Reply:
I've changed it to "Serial forward failed"